### ISSUE TITLE
tarantoolctl: start admin console BEFORE calling box.cfg()

### ISSUE THREAD

**Issue by rtsisyk** [tarantool/tarantool#2079]
It would be nice to have working admin console before starting replication. For example, I created  `example.lua` multi-master configuration, saved it to `/etc/tarantool/instance.enabled` and typed `systemctl start tarantool@example`. My instance has started, I can see "active(running)" in `systemctl status tarantool@example`, but `tarantoolctl enter example` doesn't work until multi-master is configured.

**Comment by rtsisyk** [comment]
1.7.4-318-gb1cb4a1f3
